recipes/kibana/Dockerfile

```diff
@@ -0,0 +1,36 @@
+
```

Comment:
This Dockerfile recipe fetches a distribution tarball of Kibana hosted on Google Drive. I am not sure this is what we want to do.